Tests/js/views/assets/ez-editpreviewview-tests.js

```diff
@@ -20,7 +20,7 @@ YUI.add('ez-editpreviewview-tests', function (Y) {
             });
             this.mockVersion = new Y.eZ.Version({
                 versionNo: this.versionNo,
-                names: {value: [{'_languageCode': 'eng-GB', '#text': 'Test name'}]}
+                names: {value: [{'_languageCode': 'eng-GB', '#text': 'Test name'}, {'_languageCode': 'quenya', '#text': 'Test name'}]}
```

Comment:
that's not enough in terms of test. First you should put a different value than the one for eng-GB and then in a test (""Test available variable in template" I guess), you should assert that the template receives the expected name.